cli/flox-rust-sdk/src/providers/container_builder.rs

```diff
@@ -22,6 +23,7 @@ pub trait ContainerBuilder {
     type Error: std::error::Error;
     fn create_container_source(
         &self,
+        flox: &Flox,
```

Comment:
opinion (nb): I still think how this is used it shouldn't be part of the interface for "creating a container", but rather be part of the configuration of the implementation. If we gernally want to stick with a general context object that we can pass everywhere, that's fine too, but in that case we should work on splitting context from dependency injection, or we create a maze of implicitly dependent providers.

Reply:
+1, could only pass `verbosity: i32` for now.